conf/messages.en

```diff
@@ -182,6 +183,14 @@ previousCommodityCode.error.previousCommodityCode.length = Previous Tariff Infor
 previousCommodityCode.details.summary = What is my Binding Tariff Information ruling reference number?
 previousCommodityCode.details.contentParagraph = The reference number was supplied on the certificate when you were given your original ruling.
 
+provideGoodsName.title = Provide a name for the goods - Manage your Advance Tariff Rulings - GOV.UK
+provideGoodsName.heading = Provide a name for the goods
+provideGoodsName.preheader.hint = About the goods
```

Comment:
I suggest we call this `caption` so we are consistent with the change @DavidGregory084 made here
https://github.com/hmrc/binding-tariff-trader-frontend/commit/e4c8b462b7c2dc663913f4dd784d6d781de82e28

Reply:
It has been addressed! 👍